src/modules/user/openOrders/helpers.ts

```diff
@@ -100,5 +100,5 @@ export const insertIfNotExisted = (list: OrderCommon[], order: OrderCommon): Ord
     const index = list.findIndex((value: OrderCommon) =>
         order.confirmed ? value.id === order.id : value.uuid === order.uuid);
 
-    return (index === -1) ? list.concat({...order}) : [...list];
+    return (index === -1) ? [{...order}, ...list] : [...list];
```

Comment:
Does it okay with pagination?
If there are 10 orders and user creates 1, so total is changed also

Reply:
there is no pagination in this module
Open Orders table with pagination is fetching from ordersHistory

Reply:
Okay